time stamp
string source_node
string destination_node
string connection_id
string source_topic
string destination_topic

float32 messages_per_second
float32 message_bytes_per_second

float32 average_fragment_count

float32 packet_bytes_per_second
float32 ok_sent_bytes_per_second
float32 failed_sent_bytes_per_second
float32 dropped_bytes_per_second
